.github/RELEASE.md

```diff
@@ -0,0 +1,265 @@
+<!-- These docs are updated and synced from https://github.com/conda/infra -->
+
+## Release Process
+
+[epic template]: ../../issues/new?assignees=&labels=epic&template=epic.yml
+[rever docs]: https://regro.github.io/rever-docs
+[compare]: ../../compare
+[release notes]: ../../releases/new
+
+### 1. Open an issue in the applicable project for the upcoming release with the following details.
+
+> **Note:**
+> The [epic template][epic template] is perfect for this, just remember to remove the https://github.com/conda/infra/labels/epic label.
+
+<details>
+<summary><code>GitHub Issue Template</code></summary>
+
+```markdown
+### Summary
+
+Placeholder for `conda 22.9.0` release.
+
+### Tasks
+
+[milestone]: https://github.com/conda/conda/milestone/56
+[releases]: https://github.com/conda/conda/releases
+[main]: https://github.com/AnacondaRecipes/conda-feedstock
+[conda-forge]: https://github.com/conda-forge/conda-feedstock
+
+- [ ] [Complete outstanding PRs][milestone]
+- [ ] Create release PR
+    - See release process https://github.com/conda/infra/issues/541
+- [ ] [Publish Release][releases]
+- [ ] Create/update `22.9.x` branch
+- [ ] Feedstocks
+    - [ ] Bump version [Anaconda's main][main]
+    - [ ] Bump version [conda-forge][conda-forge]
+    - Link any other feedstock PRs that are necessary
+- [ ] Handoff to the Anaconda packaging team
```

Comment:
I wonder if we should add an item here like "interact with conda-forge maintainers" or something to make clear that bumping the version in the cf recipe alone doesn't mean it's released.

Reply:
hm ok I'll keep this in mind and workshop it in https://github.com/conda/conda/issues/11802 and https://github.com/conda/conda-package-handling/issues/120, we can add whatever we decide on in a future PR?

Reply:
https://github.com/conda/infra/issues/602

Reply:
Sounds good, thank you!